lighthouse-core/gather/gather-runner.js

```diff
@@ -298,9 +306,14 @@ class GatherRunner {
     // Example: `DNS_FAILURE` is better than `NO_FCP`.
     if (networkError) return networkError;
 
-    // Error if page is not HTML.
-    if (nonHtmlError) return nonHtmlError;
-
+    if (finalRecord?.mimeType === XHTML_MIME_TYPE) {
```

Comment:
@brendankenny I _think we can use `?.` but want to double check. Just shouldn't in the report, right?

Reply:
> I _think_ we can use `?.` but want to double check. Just shouldn't in the report, right?

not yet, we're still waiting for the switch to node 14 minimum

Reply:
(we can now use this syntax)